terraform/modules/windows-test-domain/scripts/WEC/join_domain.ps1

```diff
@@ -1,7 +1,7 @@
-$User = "shire.com\wecserver"
+$User = "$env:domain\wecserver"
 $Password = ConvertTo-SecureString -String "Edhellen$" -AsPlainText -Force
 $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $User, $Password
 
-Add-Computer -DomainName "shire.com" -OUPath "OU=Servers,DC=shire,DC=com" -Credential $Credential
+Add-Computer -DomainName "$env:domain" -OUPath "OU=Servers,DC=shire,DC=com" -Credential $Credential
```

Comment:
I think these `"OU=Servers,DC=shire,DC=com"` probably need to change but I'm not sure what to.

Reply:
ah. Well spotted.